site/content/support.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Where to Go for Support"
+draft: false
+weight: 1100
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-OOO"
+---
+
+## Support Policy
+
+NGINX Gateway Fabric adheres to the support policy detailed at [nginx.com/support](https://nginx.com/support).
+
+## Commercial Support
+
+For paid customers of NGINX Gateway Fabric, commercial support is available by opening a support case at the [MyF5](https://my.f5.com/) portal.
+
+Generate a Support Package
+To generate a support package, run the following command:
+
+- (TBD)
+
+## Community Support
+
+For NGINX Gateway Fabric support or issues not addressed by documentation, you can reach out using the following two options:
+
+- Create an issue using the [Issues tab in the NGINX Gateway Fabric GitHub repo](https://github.com/nginxinc/nginx-gateway-fabric/issues).
+
+    We use issues for bug reports and to discuss new features. Creating issues is good, creating good issues is even better. Filing meaningful bug reports with lots of information in them helps us figure out what to fix when and how it impacts our users. We like bugs because it means people are using our code, and we like fixing them even more.
+
+    All issues should follow these guidelines:
+
+    -Describe the problem. Include version of Kubernetes, NGINX Gateway Fabric version, and what Kubernetes platform.
+    -Include detailed information about how to recreate the issue.
+    -Include relevant configurations, error messages, and logs.
+    -Sanitize the data. For example, be mindful of IPs, ports, application names, and URLs.
```

Comment:
These guidelines are solid and I think should be moved farther up in the doc. This is information anyone would need, whether they were a commercial support engineer, someone looking through the repository or posting on Slack.

It provides a guardrail on how to effectively communicate, so giving it more prominence could shape how everyone engages.

Reply:
I removed the guidelines based on the fact that the template in GitHub already includes them